events/2022-moveit-community-meeting/index.markdown

```diff
@@ -0,0 +1,47 @@
+---
+author: Sean Kolk
+comments: false
+date: 2022-01-10 10:00:00+00:00
+layout: page
+slug: 2022-moveit-community-meeting
+title: 2022 MoveIt Community Meeting
+---
+
+## 2022 MoveIt Community Meeting
+The 2022 Moveit Community is a time for the maintainers, enthusiasts, and experimenters with MOveit to meet, discuss, and decide the future of MoveIt. The event will be 5 hours with speakers, decision groups, and exhibitions to show MoveIt projects to the community. The event will be held virtually, with details to be determined.
+
+
+### Time and Venue
+
+* Date: Thursday, Feb 24, 2022
+* Venue: (virtual)
+
+### Organizers
+
+* Mark Moll (PickNik)
+* Henning Kayser (PickNik)
+* Tyler Weaver (PickNik)
+
+
+### Program (Tentative)
+
+All times are UTC-6 (US Mountain  Time)
+
+| Opening                             | 08:00:00    |
```

Comment:
I'm not sure about this agenda, does it work with the expected number of speakers? @mamoll any thoughts?

Reply:
I'd reduce the total meeting time and have more & shorter talks.